metadata.json

```diff
@@ -60,6 +70,6 @@
     }
   ],
   "dependencies": [
-  
+
```

Comment:
This needs the whitespace to match the checksum for the generated metadata.json from `puppet module build`